packages/daemon/src/app/controller/base.ts

```diff
@@ -33,6 +33,9 @@ export class BaseEventController extends BaseController {
       logStream.on('error', err => {
         sse.emit('error', err.message);
       });
+      logStream.on('jobStatusChange', data => {
```

Comment:
This event name looks weird, the log shouldn't be related to the job.

Reply:
Updated.